Task4/index.html

```diff
@@ -0,0 +1,41 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+	<link rel="stylesheet" href="style.css">
```

Comment:
You're using spaces everywhere but in this line tab is used. You can use tabs or spaces for indents, but don't mix it. By the way, spaces are more conventional neither that tabs.

```suggestion
        <link rel="stylesheet" href="style.css">
```